pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -520,7 +520,20 @@ https://github.com/lsig/messenger.nvim/,HEAD,
 https://github.com/xero/miasma.nvim/,,
 https://github.com/dasupradyumna/midnight.nvim/,,
 https://github.com/phaazon/mind.nvim/,HEAD,
+https://github.com/echasnovski/mini.ai/,HEAD,
+https://github.com/echasnovski/mini.animate/,HEAD,
+https://github.com/echasnovski/mini.comment/,HEAD,
+https://github.com/echasnovski/mini.completion/,HEAD,
+https://github.com/echasnovski/mini.diff/,HEAD,
+https://github.com/echasnovski/mini.files/,HEAD,
+https://github.com/echasnovski/mini.hipatterns/,HEAD,
+https://github.com/echasnovski/mini.icons/,HEAD,
+https://github.com/echasnovski/mini.indentscope/,HEAD,
+https://github.com/echasnovski/mini.move/,HEAD,
 https://github.com/echasnovski/mini.nvim/,,
+https://github.com/echasnovski/mini.pairs/,HEAD,
+https://github.com/echasnovski/mini.starter/,HEAD,
+https://github.com/echasnovski/mini.surround/,HEAD,
```

Comment:
There are over 40 `mini.*` plugins, why not add all of them then?

![image](https://github.com/user-attachments/assets/13897d67-f9ee-4bd8-932a-aa250af99806)

Reply:
✅ Done!